lib/warbler-pre-release-tillsc.rb

```diff
@@ -0,0 +1 @@
+require 'warbler'
```

Comment:
I guess this wasn't meant to go into the commit, but was a way of testing out the change locally.

Reply:
You're absolutely right. Sorry... Is't a bit hard to test since the :git bundler dependencies aren't working...